src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/update/actions/SkipPathsAndResourcesDeallocationAction.java

```diff
@@ -59,8 +60,4 @@ public void perform(State from, State to, Event event, FlowUpdateContext context
             stateMachine.fire(Event.UPDATE_ENDPOINT_RULES_ONLY);
         }
     }
-
-    private FlowPath getFlowPath(PathId pathId) {
```

Comment:
This method, which does not throw exceptions if the path is not found, is written here for a reason. Do not change logic without a good reason.

Reply:
Didn't notice, thanks